### PR TITLE
Modify translation for disable_loop_pipelining metadata

### PR DESCRIPTION
**Pull request by pauzinl**
This patch adds translation for disable_loop_pipelining metadata with value of 0

### CODE REVIEW

**Comment by pauzinl** [comment]
Hi, @MrSidims, please take a look

File: test/transcoding/SPV_INTEL_kernel_attributes/intel_fpga_function_attributes.ll [hunk 1]
```diff
@@ -53,6 +53,7 @@
 ; CHECK-SPIRV-DAG: Decorate [[FUNCENTRY]] InitiationIntervalINTEL 10
 ; CHECK-SPIRV-DAG: Decorate [[FUNCENTRY]] MaxConcurrencyINTEL 12
 ; CHECK-SPIRV-DAG: Decorate [[FUNCENTRY]] PipelineEnableINTEL 0
+; CHECK-SPIRV: Decorate [[FUNCENTRY2:[0-9]+]] PipelineEnableINTEL 1
```
Comment (MrSidims) [inline test/transcoding/SPV_INTEL_kernel_attributes/intel_fpga_function_attributes.ll:56]: FUNCENTRY2 is not used later, should it be added to check, that decoration is applied to the correct entry point?

Comment (pauzinl) [reply test/transcoding/SPV_INTEL_kernel_attributes/intel_fpga_function_attributes.ll]: Added it in [a75b8f7](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1323/commits/a75b8f7e4fd53f9d10dd9e19810dd3b5bb0e1ff8)

File: test/transcoding/SPV_INTEL_kernel_attributes/intel_fpga_function_attributes.ll [hunk 2]
```diff
@@ -43,6 +43,7 @@
 ; CHECK-SPIRV: Extension "SPV_INTEL_fpga_cluster_attributes"
 ; CHECK-SPIRV: Extension "SPV_INTEL_fpga_invocation_pipelining_attributes"
 ; CHECK-SPIRV: Extension "SPV_INTEL_loop_fuse"
+; CHECK-SPIRV: EntryPoint 6 [[FUNCENTRY2:[0-9]+]] "_ZTSZ3barvE11kernel_name3"
```
Comment (MrSidims) [inline test/transcoding/SPV_INTEL_kernel_attributes/intel_fpga_function_attributes.ll:46]:
```suggestion
; CHECK-SPIRV: EntryPoint {{.*}} [[FUNCENTRY2:[0-9]+]] "_ZTSZ3barvE11kernel_name3"
```

**Review by MrSidims — APPROVED** [review]

**Pull request merged**